### ISSUE TITLE
comm creation UX

### ISSUE THREAD

**Issue by roadscape** [steemit/wallet#106]
- form UX needs usability sweep & polishing
- add a subscription op (creator to subscribe to created community)

**Comment by gl2748** [comment]
```
1/ please remove “Is not safe for work” checkbox, and do not broadcast this value
2/ rename “Description” to “About”
3/ style “owner name” and “password” lines — owner name can be in a <code> tag, password should be styled
4/ what do you think about adding a password confirm box?
5/ this box should say something like “This operation will cost 3 STEEM. Would you like to proceed?”
6/ when I click “Cancel” on the confirm dialog, I’m sent back to the form with a confusing error: “Unable to create that community. Please ensure you used the correct key.” (check if there’s an easy way to detect cancel vs. error — if not.. don’t spend too much time on it)
7/ styling of “in progress” messages as well as error messages — the plain text messages are hard to parse. any sort of basic styling or showing of progress would be a big help
```

**Comment by roadscape** [comment]
#107 merged
